content/docs/develop/reference/bridge/basic_authentication/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Basic Authentication
+description: Enable/Disable basic authentication
+weight: 21
+keywords: [bridge]
+---
+
+The Keptn Bridge has a basic authentication feature, which can be controlled by setting the following two environment variables:
+
+* `BASIC_AUTH_USERNAME` - username
+* `BASIC_AUTH_PASSWORD` - password
+
+## Enable Authentication
+
+To enable this feature, a secret has to be created that holds the two variables. This secret has to be applied within the Kubernetes deployment for the Keptn Bridge.
```

Comment:
With Keptn 0.7, this secret is automatically created by the `api-service` when exposing the bridge. Can we please add this info. Otherwise, one could assume that this has to be done manually.

Reply:
Valid point. I explained the new way of exposing the Bridge that automatically creates the secret.

Reply:
Thanks!